swap_meet/item.py

```diff
@@ -0,0 +1,27 @@
+class Item:
+    def __init__(self, category='', condition=0):
+        # if category is None:
+        #     self.category = ""
+        # else:
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+        # return self.category
+
+    def condition_description(self):
+        description = ""
+        if self.condition == 0:
+            description = "Seems uselesss, unless ..."
+        elif self.condition == 1:
+            description = "Happy if you can give it a try!"
+        elif self.condition == 2:
+            description = "Used - Fair"
+        elif self.condition == 3:
+            description = "Used - Good"
+        elif self.condition == 4:
+            description = "Used - Like new"
+        elif self.condition == 5:
+            description = "New"
+        return description
```

Comment:
I like your condition descriptions. It seems `condition` is expected to be an integer, adding type checking would be really useful here.

Reply:
I have added validation of `condition` to the `__init__` function of the class. Is that the right thing to do? 

```
class Item:
    def __init__(self, category='', condition=0):
        self.category = category
        if isinstance(condition, float) or isinstance(condition, int):
            self.condition = condition
        else:
            raise Exception("A condition is not a number.")
```

Reply:
I also changed the function `condition_description` to check not for the strict equality but for the presence in a given range.  
```
def condition_description(self):
        description = ""
        if 0.0 <= self.condition < 1.0:
            description = "Seems uselesss, unless ..."
        elif 1.0 <= self.condition < 2.0:
            description = "Happy if you can give it a try!"
        elif 2 <= self.condition < 3.0:
            description = "Used - Fair"
        elif 3 <= self.condition < 4.0:
            description = "Used - Good"
        elif 4 <= self.condition < 5.0:
            description = "Used - Like new"
        else:
            description = "Brand New"
        return description
```